scala-xml/src/main/scala/org/http4s/scalaxml/ElemInstances.scala

```diff
@@ -49,7 +52,8 @@ trait ElemInstances {
     *
     * @return an XML element
     */
-  implicit def xml[F[_]](implicit F: Concurrent[F]): EntityDecoder[F, Elem] = {
+  @deprecated("Blocks. Use xmlDecoder with an Async constraint.", "0.23.12")
+  def xml[F[_]](implicit F: Concurrent[F]): EntityDecoder[F, Elem] = {
```

Comment:
If we care enough we can do a runtime check for `Async` to delegate to the new implementation.

Reply:
That technique makes me squeamish, but we've adopted it enough other places and I've never seen an accident scene from it.

Reply:
Oops.  I merged before resolving this.  If we want, it could still be a separate PR.